backend/windmill-api/openapi.yaml

```diff
@@ -2086,6 +2085,36 @@ paths:
             application/json:
               schema: {}
 
+  /w/{workspace}/jobs/result_by_id/{flow_job_id}/{node_id}:
+    get:
+      summary: get job result by id
+      operationId: resultById
+      tags:
+        - job
+      parameters:
+        - $ref: "#/components/parameters/WorkspaceId"
+        - name: flow_job_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: node_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: skip_direct
+          description: TODO Add description, I'm not even really sure what this does
+          in: query
+          schema:
+            type: boolean
+      responses:
+        "200":
+          description: job result
+          content:
+            application/json:
+              schema: {}
+
```

Comment:
Take note of this. I've never used a large OpenAPI document like this, so not sure if correct like this, or if things should be factored out into #/... references.
Also the description.

Reply:
The rule of 3, references that would only be used once or twice are not worth being made in most cases